wordpress-mobile/WordPress-iOS
# Integrate Aztec 1.0.0-beta.13 into WPiOS.

### Description:

Integrate Aztec 1.0.0-beta.13 into WPiOS.

A new pod version will be released once this PR is approved.

### Things to test:

- Spacing between paragraphs, lines of text in the same paragraph (shift + ENTER), list items, block quotes, list with block quotes.  Switch to HTML and back to ensure spacing is not lost.
- Drag and drop form other Apps (such as Safari).
- Copy & paste.  Also try killing the app between the copy and the paste operation.
- Backspacing should not lose any styling.
